Create fz.json

masfaiz-code opened the pull request:
<!--
YOU MUST FILL OUT THIS TEMPLATE ENTIRELY FOR YOUR PR TO BE ACCEPTED, NONE OF IT IS OPTIONAL UNLESS SPECIFIED.
-->

# Requirements
<!-- Your domain MUST pass ALL the requirements below, otherwise it WILL BE DENIED. -->

<!-- Change each checkbox to [x] (all lowercase, with no spaces between the brackets) to mark it as completed. -->

- [x] I have **read** and **agreed** to the [Terms of Service](https://is-a.dev/terms). <!-- Your request MUST follow the TOS to be approved. -->
- [x] My file is following the [domain structure](https://docs.is-a.dev/domain-structure/).
- [x] My website is **reachable** and **completed**. <!-- We do not permit simple "Hello, world!" or simply copied/mostly blank templated websites. -->
- [x] My website is **software development** related. <!-- Only your root subdomain needs to meet this requirement. -->
- [x] My website is not for commercial use. <!-- Your website's purpose should not be to generate any form of revenue. -->
- [x] I have provided sufficient contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g. X/Twitter or Discord) for contact. -->
- [x] I have provided a preview of my website below. <!-- This step is required for your domain to be approved. -->

# Website Preview
<!-- Provide a link or screenshot of your website below. -->

masfaiz-code committed:
Create fz.json

masfaiz-code committed:
Merge branch 'main' into main

masfaiz-code committed:
Update fz.json

masfaiz-code committed:
Delete domains/fz.json

masfaiz-code committed:
Create fz.json

masfaiz-code committed:
Merge branch 'main' into main

masfaiz-code committed:
Merge branch 'main' into main

STICKnoLOGIC reviewed (changes requested):
Add a little "About me" (yourself, tech, skill, stack, etc)

masfaiz-code commented:
Hi, I've updated my website and added a dedicated "About Me" page here:
https://bio.faa.my.id/about

It now includes my bio, skills, and tech stack.
Please recheck. Thank you!

masfaiz-code committed:
Merge branch 'main' into main

masfaiz-code committed:
Merge branch 'main' into main

STICKnoLOGIC reviewed (approved):
lgtm
